params/precompile_config.go

```diff
@@ -320,13 +342,18 @@ func (c *ChainConfig) CheckConfigurePrecompiles(parentTimestamp *big.Int, blockC
 			// If this transition activates the upgrade, configure the stateful precompile.
 			// (or deconfigure it if it is being disabled.)
 			if config.IsDisabled() {
+				log.Info("Disabling precompile", "name", key)
 				statedb.Suicide(config.Address())
 				// Calling Finalise here effectively commits Suicide call and wipes the contract state.
 				// This enables re-configuration of the same contract state in the same block.
 				// Without an immediate Finalise call after the Suicide, a reconfigured precompiled state can be wiped out
 				// since Suicide will be committed after the reconfiguration.
 				statedb.Finalise(true)
 			} else {
+				// TODO: very janky solution, instead implement stringer in the interface
+				// currently skipping this so we don't break current precompiles
+				configJSON, _ := json.Marshal(config)
+				log.Info("Activating new precompile", "name", key, "config", string(configJSON))
```

Comment:
I think it would be cleaner to simply add a log within the `Configure` function of each of the precompiles.

Reply:
I'd like to keep them in a more common place, also we cannot log from `Configure` when they're disabled. Instead we should implement `fmt.Stringer`.

Reply:
I implemented the stringer interface and simply returned json representation of configs

Reply:
Ah good point, you're 100% right